### DISCUSSION TITLE
Battle mode

### DISCUSSION THREAD

**Discussion by Szymonkruk12** [FunkinCrew/Funkin#3751] [Ideas]
Please add battle mode to friday night funkin because Playing songs in this game is very boring but battle mode there will be fun also i mean the unused battle mode where boyfriend uses the unused animations like dodge, hurt, pre-attack and attack and also where daddy dearset uses his kick and hurt and there are some blood splash when getting hurt

**Comment by Beelijah** [comment]
Do you mean 2 player?

**Comment by charlesisfeline** [comment]
i heard a few times that the full game would come with battle mode (heck, theres unused sprites for it) so maybe


if youre talking about multiplayer, yea sure i might eventually come into the game soon since there was a goal for it in the kickstarter

the thing is tho, theres a reason why both will only make it to the full game

**Comment by KoloInDaCrib** [comment]
Not sure how that would work for songs like Blazin', where the opponent side has marginally less notes than the player side to the point of having the opponent strumline hidden throughout the song

Besides I don't think any sort of multiplayer would be added soon as it requires a LOT of programming, testing, conceptualizing and so on to be of the highest priority

**Reply by PSI3x** [reply]
The erect songs would also be very awkward to play with two players with the disproportionate amount of notes. 
For statistics: Nightmare Blammed, one of the more balanced songs, has 253 to 392 notes: meaning that BF has 1.5x the amount of notes as Pico. 
In the case of Nightmare Pico, the proportion is 249 to 573: The player has 2.5x the amount of notes as the opponent.
Nightmare Spookeez, the most extreme example, has ~300 to **over 900**: a 3 to 1 ratio weighed towards the player side. 
I don't know if the two-player mode for erect songs would use a custom chart that duets the denser parts, but I see this being very awkward in multiplayer...

**Reply by Beelijah** [reply]
true PSI3x, but they could just base it on accuracy.

**Comment by PSI3x** [comment]
You really should proof-read your writing before posting it, this makes zero sense.

**Comment by Szymonkruk12** [comment]
also i mean the unused battle mode where boyfriend uses the unused animations like dodge, hurt, pre-attack and attack and also where daddy dearset uses his kick and hurt and there are some blood splash when getting hurt

**Reply by charlesisfeline** [reply]
thats likely coming to the full Steam release

**Reply by Beelijah** [reply]
tru
